third_party/move/move-compiler-v2/tests/checking/typing/loop_body_invalid.exp

```diff
@@ -18,11 +18,11 @@ error: expected expression with no value but found `address`
 11 │         loop { @0x0 }
    │                ^^^^
 
-error: expected expression with no value but found `integer`
-   ┌─ tests/checking/typing/loop_body_invalid.move:15:27
+error: tuple type `()` is not allowed
```

Comment:
Error message needs to be improved.

Reply:
Please file a bug or add this case to an existing bug, if you have not already.

Reply:
Will file the bug once the PR is landed